From ba92e7d6edda544971d563a77ba787c13e05e023 Mon Sep 17 00:00:00 2001
From: "Loc Phuoc Pham.Lam" <loc.lam.px@renesas.com>
Date: Tue, 10 Sep 2019 11:38:55 +0700
Subject: [PATCH] plat: renesas: rcar: qos: M3N: mstat195.h: Modify default
 setting for RZ/G2N

This patch file disables 8 unused modules:
 + IMR
 + RealTime Core
 + RT-DMAC
 + Secure Engine
 + ADSP
 + DCU
 + SSP
 + DAB

Signed-off-by: Loc Phuoc Pham.Lam <loc.lam.px@renesas.com>
---
 .../rcar/qos/M3N/qos_init_m3n_v10_mstat195.h       | 32 +++++++++++-----------
 1 file changed, 16 insertions(+), 16 deletions(-)

diff --git a/plat/renesas/rcar/qos/M3N/qos_init_m3n_v10_mstat195.h b/plat/renesas/rcar/qos/M3N/qos_init_m3n_v10_mstat195.h
index eb8bced..31c62b9 100644
--- a/plat/renesas/rcar/qos/M3N/qos_init_m3n_v10_mstat195.h
+++ b/plat/renesas/rcar/qos/M3N/qos_init_m3n_v10_mstat195.h
@@ -29,11 +29,11 @@ static uint64_t mstat_fix[]={
 	/* 0x00d8, */ 0x0000000000000000UL,
 	/* 0x00e0, */ 0x0000000000000000UL,
 	/* 0x00e8, */ 0x0000000000000000UL,
-	/* 0x00f0, */ 0x001024840000FFFFUL,
+	/* 0x00f0, */ 0x0000000000000000UL,
 	/* 0x00f8, */ 0x000C084F0000FFFFUL,
 	/* 0x0100, */ 0x0000000000000000UL,
 	/* 0x0108, */ 0x0000000000000000UL,
-	/* 0x0110, */ 0x00100C840000FFFFUL,
+	/* 0x0110, */ 0x0000000000000000UL,
 	/* 0x0118, */ 0x000C21E60000FFFFUL,
 	/* 0x0120, */ 0x0000000000000000UL,
 	/* 0x0128, */ 0x0000000000000000UL,
@@ -60,12 +60,12 @@ static uint64_t mstat_fix[]={
 	/* 0x01d0, */ 0x0000000000000000UL,
 	/* 0x01d8, */ 0x0000000000000000UL,
 	/* 0x01e0, */ 0x0000000000000000UL,
-	/* 0x01e8, */ 0x000C04010000FFFFUL,
-	/* 0x01f0, */ 0x000C04050000FFFFUL,
+	/* 0x01e8, */ 0x0000000000000000UL,
+	/* 0x01f0, */ 0x0000000000000000UL,
 	/* 0x01f8, */ 0x0000000000000000UL,
 	/* 0x0200, */ 0x0000000000000000UL,
-	/* 0x0208, */ 0x000C04050000FFFFUL,
-	/* 0x0210, */ 0x000C04050000FFFFUL,
+	/* 0x0208, */ 0x0000000000000000UL,
+	/* 0x0210, */ 0x0000000000000000UL,
 	/* 0x0218, */ 0x0000000000000000UL,
 	/* 0x0220, */ 0x0000000000000000UL,
 	/* 0x0228, */ 0x0000000000000000UL,
@@ -75,24 +75,24 @@ static uint64_t mstat_fix[]={
 	/* 0x0248, */ 0x0000000000000000UL,
 	/* 0x0250, */ 0x0000000000000000UL,
 	/* 0x0258, */ 0x0000000000000000UL,
-	/* 0x0260, */ 0x000C08150000FFFFUL,
+	/* 0x0260, */ 0x0000000000000000UL,
 	/* 0x0268, */ 0x001408020000FFFFUL,
 	/* 0x0270, */ 0x001404010000FFFFUL,
-	/* 0x0278, */ 0x000C04090000FFFFUL,
+	/* 0x0278, */ 0x0000000000000000UL,
 	/* 0x0280, */ 0x0000000000000000UL,
 	/* 0x0288, */ 0x0000000000000000UL,
 	/* 0x0290, */ 0x001408020000FFFFUL,
 	/* 0x0298, */ 0x001404010000FFFFUL,
 	/* 0x02a0, */ 0x000C04050000FFFFUL,
 	/* 0x02a8, */ 0x000C04050000FFFFUL,
-	/* 0x02b0, */ 0x001408050000FFFFUL,
-	/* 0x02b8, */ 0x000C04010000FFFFUL,
+	/* 0x02b0, */ 0x0000000000000000UL,
+	/* 0x02b8, */ 0x0000000000000000UL,
 	/* 0x02c0, */ 0x0000000000000000UL,
 	/* 0x02c8, */ 0x0000000000000000UL,
 	/* 0x02d0, */ 0x000C04050000FFFFUL,
 	/* 0x02d8, */ 0x000C04050000FFFFUL,
-	/* 0x02e0, */ 0x001408050000FFFFUL,
-	/* 0x02e8, */ 0x000C04010000FFFFUL,
+	/* 0x02e0, */ 0x0000000000000000UL,
+	/* 0x02e8, */ 0x0000000000000000UL,
 	/* 0x02f0, */ 0x0000000000000000UL,
 	/* 0x02f8, */ 0x0000000000000000UL,
 	/* 0x0300, */ 0x0000000000000000UL,
@@ -109,8 +109,8 @@ static uint64_t mstat_fix[]={
 	/* 0x0358, */ 0x0000000000000000UL,
 	/* 0x0360, */ 0x0000000000000000UL,
 	/* 0x0368, */ 0x0000000000000000UL,
-	/* 0x0370, */ 0x000C04010000FFFFUL,
-	/* 0x0378, */ 0x000C04010000FFFFUL,
+	/* 0x0370, */ 0x0000000000000000UL,
+	/* 0x0378, */ 0x0000000000000000UL,
 	/* 0x0380, */ 0x000C04050000FFFFUL,
 	/* 0x0388, */ 0x000C04050000FFFFUL,
 	/* 0x0390, */ 0x0000000000000000UL,
@@ -178,11 +178,11 @@ static uint64_t mstat_be[]={
 	/* 0x01d8, */ 0x0000000000000000UL,
 	/* 0x01e0, */ 0x0000000000000000UL,
 	/* 0x01e8, */ 0x0000000000000000UL,
-	/* 0x01f0, */ 0x002101000BDF2401UL,
+	/* 0x01f0, */ 0x0000000000000000UL,
 	/* 0x01f8, */ 0x0000000000000000UL,
 	/* 0x0200, */ 0x0000000000000000UL,
 	/* 0x0208, */ 0x0000000000000000UL,
-	/* 0x0210, */ 0x002101000BDF2401UL,
+	/* 0x0210, */ 0x0000000000000000UL,
 	/* 0x0218, */ 0x001101000BDF2401UL,
 	/* 0x0220, */ 0x001101000BDF2401UL,
 	/* 0x0228, */ 0x0000000000000000UL,
-- 
1.9.1

